src/KubernetesClient/Kubernetes.ConfigInit.cs

```diff
@@ -104,18 +88,18 @@ public Kubernetes(KubernetesClientConfiguration config, params DelegatingHandler
             {
                 throw new KubeConfigException("Bad host url", e);
             }
+        }
 
-            CaCerts = config.SslCaCerts;
```

Comment:
Looks to me like this code was dropped? (I could be wrong...)

Reply:
Oh, is it in the constructor above? I would just move all of that down here....

Reply:
I would, but doing so won't compile because CaCerts only has an empty getter that needs to be assigned where it is declared, or directly within a constructor.